modules/charon-core/src/main/java/org/wso2/charon3/core/objects/AbstractSCIMObject.java

```diff
@@ -46,9 +45,6 @@
     /**Collection of attributes which constitute this resource.*/
     protected Map<String, Attribute> attributeList = new HashMap<String, Attribute>();
 
-    /**List of schemas where the attributes of this resource, are defined.*/
-    protected List<String> schemaList = new ArrayList<String>();
```

Comment:
It is not recommended to remove protected/public variables. Those variables might be already used.